### PR TITLE
Standardize service pom file and remove unused dependencies

### PR DESCRIPTION
**Pull request by xieus**
This PR proposes the following changes:

* Standardize project metadata (including groundID, artifactId, name, version) in pom files of various services, lib, web and schema.

* Remove unused dependencies and plugin in pom files

* Update dockerfiles of various services.

### CODE REVIEW

**Comment by kimeunju108** [comment]
LGTM

File: services/mac_manager/pom.xml [hunk 1]
```diff
@@ -32,58 +32,28 @@
...
-        <dependency>
-            <groupId>org.apache.ignite</groupId>
-            <artifactId>ignite-core</artifactId>
-            <version>2.8.0</version>
-        </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
-            <exclusions>
```
Comment (xieus) [inline services/mac_manager/pom.xml:80]: @kimeunju108 Could you review this part to see which lib is needed. Currently the mac_manager pom change caused UT failure: https://jenkins.alkaidcloud.io/job/alcor-controller-pr-test/372/console

Comment (xieus) [reply services/mac_manager/pom.xml:80]: If we add org.junit.vintage back, MacManager basically won't run any UTs.

Comment (xieus) [reply services/mac_manager/pom.xml]: @kimeunju108, as this PR include other important payload, I will temporarily disable UTs for mac manager and go ahead to merge the PR. 

Created an issue for tracking: https://github.com/futurewei-cloud/alcor/issues/199

Please fix the UTs with priority. A service can't sustain for long without active UT running.

**Pull request merged**